Makefile

```diff
@@ -4,7 +4,7 @@
 NAME?=elys
 BINARY?=$(NAME)d
 COMMIT:=$(shell git log -1 --format='%H')
-VERSION:=$(shell git describe --tags --match 'v*' --abbrev=8 | sed 's/-g/-/' | sed 's/-[0-9]*-/-/')
+VERSION:=v999.999.999
```

Comment:
Why overwriting this change. This is used -X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \ which goes into binary and upgrades are based on this.